.travis/analyse.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+function finish() {
+  make logs stop
+}
+
+trap finish EXIT
+
+make start wait-healthy
```

Comment:
only thing to note was that this failed rather noisily when I was missing the submodule:
```
[14:06:04][giorgio@Newton:~/code/article-store]$ .travis/analyse.sh                                                    [98/251]
docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.dev.yml up --detach
Creating network "docker_default" with the default driver
Creating docker_app_1 ... done
Creating docker_console_1 ... done
services=($(docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.dev.yml ps --quiet)); \
if [ ${#services[@]} -eq 0 ]; then \
        echo "No containers running"; \
        exit 1; \
fi; \
for service in ${services}; do \
        .scripts/docker/wait-healthy.sh "${service}"; \
done
bash: line 6: .scripts/docker/wait-healthy.sh: No such file or directory
Makefile:34: recipe for target 'wait-healthy' failed
make: *** [wait-healthy] Error 127
docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.dev.yml logs
Attaching to docker_console_1, docker_app_1
app_1      | 
app_1      | > @ start:dev /app
app_1      | > nodemon --exec 'node --require ts-node/register ./src/index.ts' --ext .js,.json,.ts
app_1      | 
app_1      | [nodemon] 1.19.4
app_1      | [nodemon] to restart at any time, enter `rs`
app_1      | [nodemon] watching dir(s): *.*
app_1      | [nodemon] watching extensions: js,json,ts
app_1      | [nodemon] starting `node --require ts-node/register ./src/index.ts`
console_1  | Listening on http://68c32a97136c:8000, http://127.0.0.1:8000, http://192.168.144.3:8000
docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.dev.yml down
Stopping docker_console_1 ... 
Stopping docker_console_1 ... done
Stopping docker_app_1     ... done
Removing docker_console_1 ... done
Removing docker_app_1     ... done
Removing network docker_default
```
I wouldn't go as far as adding git commands to the Makefile, because CI systems like Travis or Jenkins take care of this automatically. But noting down a ` git submodule update -i` in the README could help. Once the submodule is initialized `git diff` will show updates so it will be impossible to miss.

Reply:
Probably should be in `make install` though.